testing/ef_tests/src/cases/operations.rs

```diff
@@ -122,8 +122,15 @@ impl<E: EthSpec> Operation<E> for AttesterSlashing<E> {
         "attester_slashing".into()
     }
 
-    fn decode(path: &Path, _fork_name: ForkName, _spec: &ChainSpec) -> Result<Self, Error> {
-        ssz_decode_file(path)
+    fn decode(path: &Path, fork_name: ForkName, _spec: &ChainSpec) -> Result<Self, Error> {
+        Ok(match fork_name {
+            ForkName::Base
+            | ForkName::Altair
+            | ForkName::Bellatrix
+            | ForkName::Capella
+            | ForkName::Deneb => Self::Base(ssz_decode_file(path)?),
+            ForkName::Electra => Self::Electra(ssz_decode_file(path)?),
+        })
     }
```

Comment:
something is strange here.. the EF `attester_slashing` tests pass even when I make this always decode `Electra`

Reply:
The only difference is on the merkleization right, operation tests only SSZ deserialize and test functionality.